web/src/js/components/contentviews/HttpMessage.tsx

```diff
@@ -32,7 +33,11 @@ export default function HttpMessage({ flow, message }: HttpMessageProps) {
         [maxLines],
     );
     const [edit, setEdit] = useState<boolean>(false);
-    let url;
+    const [isCopied, setIsCopied] = useState<boolean>(false);
```

Comment:
I don't mean to be super nit-picky. Just wondering if there's any naming convention in the rest of the project. `edit` vs `isEditing` and `copied` vs `isCopied`.

Reply:
Good point! Normally, if it's a boolean value, we use the prefix `is`. For this reason, I've adjusted the `edit` value to `isEdited`